### PR TITLE
[SYCL] [L0] Correct the device id check for PVC.

### PR DESCRIPTION
**Pull request by rdeodhar**
Account for the various device ids used for PVC GPUs.

### CODE REVIEW

File: sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp [hunk 1]
```diff
@@ -182,7 +182,10 @@ struct _ur_device_handle_t : _ur_object {
   bool isSubDevice() { return RootDevice != nullptr; }
 
   // Is this a Data Center GPU Max series (aka PVC).
-  bool isPVC() { return (ZeDeviceProperties->deviceId & 0xff0) == 0xbd0; }
+  bool isPVC() {
+    return (ZeDeviceProperties->deviceId & 0xff0) == 0xbd0 ||
+           (ZeDeviceProperties->deviceId & 0xff0) == 0xb60;
```
[the remark this answers is not in the document]
Comment (rdeodhar) [reply sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp:187]: The device ip version suffers from a similar issue where the versions range from 12.60.0 to 12.60.7 which in hex are 0x030f0000 to 0x030f0007 with no guarantee that the numbers won't keep rising. Also there is added complexity in getting to the device extension struct. All in all, there seems no advantage to using the device ip version.

[the remark this answers is not in the document]
Comment (jandres742) [reply sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp:187]: @smaslov-intel : we could use either that or this one, I dont think there's any major advantage of one vs the other. in terms of device ids, this won't change too much, so maybe we would have to add new device ids if they appear, but that would be sporadically, and is a more stable interface. device-ip was just added recently and information about versioning is not widely publicly available.

I would suggest to stay with the device id for now, and we can transition to device ip gradually in the future when it starts to be more widely used.

[the remark this answers is not in the document]
Comment (smaslov-intel) [reply sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp:187]: OK

**Review by smaslov-intel — APPROVED** [review]

File: sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (smaslov-intel) [reply sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp:187]: Perhaps. add a TODO comment to change to use https://spec.oneapi.io/level-zero/latest/core/api.html#ze-device-ip-version-ext-t when that is stable?

**Review by smaslov-intel — APPROVED** [review]

File: sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (rdeodhar) [reply sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.hpp:190]: Ok, have added a TODO comment.

**Pull request merged**